js/utils.js

```diff
@@ -0,0 +1,57 @@
+export function getFirstDayOfTheWeek(date) {
+  const day = date.getDay();
+  const diff = date.getDate() - day + (day === 0 ? -6 : 1);
+  const startOfWeek = new Date(date.setDate(diff));
+  startOfWeek.setHours(0, 0, 0, 0);
+  return startOfWeek;
+}
+
+export function isToday(date) {
+  const today = new Date();
+  return (
+    date.getDate() === today.getDate() &&
+    date.getMonth() === today.getMonth() &&
+    date.getFullYear() === today.getFullYear()
+  );
+}
+
+export function getEventsFromLocalStorage() {
+  let events = JSON.parse(localStorage.getItem("calendarEvents")) || [];
+  return events;
+}
+
```

Comment:
these kind of functions that get or set something from database (in this case localStorage) we usually call services and they live in their own services.js file.

why it caught my attention now is because get from local storage and save in local storage are in different files. should be in same : )